### ISSUE TITLE
Move `NeetoEmailPrefixer` to `neetoCommonsBackend`

### ISSUE THREAD

**Issue by vivek-viswam-rv** [bigbinary/neeto-email-prefixer#8]
cc: @yedhink

**Comment by yedhink** [comment]
@AbhayVAshokan Can you look into whether this is feasible given that this is a forked repo? From my understanding this won't be possible if the parent is maintaining `master` branch itself.

**Comment by AbhayVAshokan** [comment]
@yedhink  It is possible to rename the default branch in the forked repo and still make a PR to either to the forked repo or to the parent repo. 

Every time we raise a new PR, we should keep in mind to properly choose the base and head branches. This would be self-explanatory.

![pr](https://user-images.githubusercontent.com/35297280/218771825-b9668d41-e1ea-4439-ad67-3546f7f3f10e.gif)

**Comment by yedhink** [comment]
@AbhayVAshokan Got it.

Another question: Why are we maintaining this as a fork but not our repo? Are we pulling updates from the parent repo from time to time? If we are pulling, then add instructions in Readme on how to pull those changes once default branch mismatch occurs between parent and child.

**Comment by AbhayVAshokan** [comment]
@yedhink the parent repo is not actively maintained. This is a small gem. To prevent any such conflicts in the future, we may extract the required portions of this gem as a concern to `neeto-commons-backend`.

**Comment by sumanranjanpanda** [comment]
Any update on this?

**Comment by AbhayVAshokan** [comment]
@sumanranjanpanda let's move this to neetoCommonsBackend. This is a very simple gem. During nv1 standardization, I verified that this is used only within neetoCommonsBackend. Hence, it should be straightforward. 

Once the migration is done, we need to delete this repository and make sure compliance is not tracking it.

Please add someone to work on it: engineer-needed _t.

cc: @yedhink

**Comment by sumanranjanpanda** [comment]
@anmol-kumar0815 _A

**Comment by anmol-kumar0815** [comment]
1.5D _t

**Comment by anmol-kumar0815** [comment]
PR for this issue has been raised. It took me longer than expected due to encountering errors while migrating this gem to neeto-commons-backend. Thus, I am updating the ETA to 2D _t.

**Comment by sumanranjanpanda** [comment]
@anmol-kumar0815 Why is this issue still pending?

**Comment by anmol-kumar0815** [comment]
@AbhayVAshokan Please review and merge this PR: https://github.com/bigbinary/neeto-commons-backend/pull/1718

**Comment by anmol-kumar0815** [comment]
[PR](https://github.com/bigbinary/neeto-commons-backend/pull/1718) merged closing this issue.

**Comment by AbhayVAshokan** [comment]
@udai1931 we have migrated the logic in this repository to neeto-commons-backend. Please remove this from compliance. LMK once this is done so that I can ping Neeraj to delete this repository.

**Comment by udai1931** [comment]
@AbhayVAshokan I have removed `neeto-email-prefixer` from compliance.

**Comment by AbhayVAshokan** [comment]
@neerajdotname this repository can be safely deleted. Please do the necessary.
